locales/app.yml

```diff
@@ -0,0 +1,528 @@
+  en: "Checking if %{path} is a git repository"
+"{path} does not exist":
+  en: "%{path} does not exist"
+"Error looking for {path}: {error}":
+  en: "Error looking for %{path}: %{error}"
+"Error getting remotes for {repo}: {error}":
+  en: "Error getting remotes for %{repo}: %{error}"
+"Skipping {path} because it's a descendant of last known repo {last_git_repo}":
+  en: "Skipping %{path} because it's a descendant of last known repo %{last_git_repo}"
+"Error in path {error}":
+  en: "Error in path %{error}"
+"Bad glob pattern: {pattern}":
+  en: "Bad glob pattern: %{pattern}"
+"Failed to pull {repo}":
+  en: "Failed to pull %{repo}"
+"Would pull {repo}":
+  en: "Would pull %{repo}"
+"Node Package Manager":
+  en: "Node Package Manager"
+"Performant Node Package Manager":
+  en: "Performant Node Package Manager"
+"Yarn is 2.x or above, skipping global upgrade":
+  en: "Yarn is 2.x or above, skipping global upgrade"
+"Yarn Package Manager":
+  en: "Yarn Package Manager"
+"Deno installed outside of .deno directory":
+  en: "Deno installed outside of .deno directory"
+"Could not find tmux":
+  en: "Could not find tmux"
+"Error determining if a tmux session exists":
+  en: "Error determining if a tmux session exists"
+"Error running Topgrade in tmux":
+  en: "Error running Topgrade in tmux"
+"Failed to compute tmux windows":
+  en: "Failed to compute tmux windows"
+"tmux session {session_name} has no windows":
+  en: "tmux session %{session_name} has no windows"
+"Toolboxes to inspect: {toolboxes}":
+  en: "Toolboxes to inspect: %{toolboxes}"
+"Wrote vim script to {path}":
+  en: "Wrote vim script to %{path}"
+"The Ultimate vimrc":
+  en: "The Ultimate vimrc"
+"vim binary might be actually nvim":
+  en: "vim binary might be actually nvim"
+"Oh-my-zsh: under SSH, setting ZSH={env_zsh}":
+  en: "Oh-my-zsh: under SSH, setting ZSH=%{env_zsh}"
+"Running zsh returned {error}. Using default path: {path}":
+  en: "Running zsh returned %{error}. Using default path: %{path}"
+"oh-my-zsh custom dir: {custom_dir}":
+  en: "oh-my-zsh custom dir: %{custom_dir}"
+"`{process}` failed: {exit_satus}":
+  en: "`%{process}` failed: %{exit_satus}"
+"Unknown Linux Distribution":
+  en: "Unknown Linux Distribution"
+'File "/etc/os-release" does not exist or is empty':
+  en: 'File "/etc/os-release" does not exist or is empty'
+"Failed getting the system package manager":
+  en: "Failed getting the system package manager"
+"A step failed":
+  en: "A step failed"
+"Dry running":
+  en: "Dry running"
+"Topgrade Upgraded":
+  en: "Topgrade Upgraded"
+"OK":
+  en: "OK"
+"FAILED":
+  en: "FAILED"
+"IGNORED":
+  en: "IGNORED"
+"SKIPPED":
+  en: "SKIPPED"
+# 'Y' and 'N' have to stay the same characters. Eg for German the translation would look sth like "(Y) Ja / (N) Nein"
+"(Y)es/(N)o":
+  en: "(Y)es/(N)o"
+# 'y', 'N', 's', 'q' have to stay the same throughout all translations. Eg German would look like "(R) Wiederholen / (N) Nein / (s) Konsole / (q) Beenden"
```

Comment:
```suggestion
# 'y', 'N', 's', 'q' have to stay the same throughout all translations. Eg German would look like "(y) Wiederholen / (N) Nein / (s) Konsole / (q) Beenden"
```

Reply:
This is no longer needed since my account has been reinstated.